tests/unit/conftest.py

```diff
@@ -171,6 +173,22 @@ def ai_getter(pyramid_config):
     return ai_getter
 
 
+@pytest.fixture
+def application_instance_service(pyramid_config):
+    application_instance_service = mock.create_autospec(
+        ApplicationInstanceService, instance=True, spec_set=True
+    )
+
+    application_instance_service.get.return_value = mock.create_autospec(
+        ApplicationInstance, consumer_key=mock.sentinel.consumer_key
+    )
```

Comment:
You need to pass `instance=True` otherwise `create_autospec` creates a mock of the _class_ rather than a mock of an instance of the class, and `spec_set=True` so that tests can't _set_ attributes on the mock that don't exist on the real object